README.md

```diff
@@ -4,7 +4,7 @@
 
 > This project was built for Microverse [under these guidance objectives](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) > _note: you can only access this link if you're a Micronault (Microverse Student)_
 
-Additional description about the project and its features.
+![screenshot](./assets/img/Project_screenshot.png)
 
 ## Built With
 
```

Comment:
- [ ] Can you kindly add instructions on how to set up the project locally in your README.md file, This will help non-technical people know how to do so, you can still refer to [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍